packages/financial-templates-lib/.mocharc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
taken from https://github.com/UMAprotocol/protocol/pull/3315 to minimize merge conflicts.